src/GitVersionTask/GitVersionTask.csproj

```diff
@@ -37,9 +38,7 @@
     <PackageReference Include="YamlDotNet" Version="$(PackageVersion_YamlDotNet)">
         <PrivateAssets>All</PrivateAssets>
     </PackageReference>
-    <!--See https://github.com/Microsoft/msbuild/issues/3671 for reason for the ExcludeAssets against msbuild packagerefernces -->
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.9.20" ExcludeAssets="runtime" />
-    <PackageReference Include="UtilPack.NuGet.MSBuild" Version="$(PackageVersion_UtilPackNuGetMSBuild)">
+    <PackageReference Include="NuGetUtils.MSBuild.Exec" Version="$(PackageVersion_UtilPackNuGetMSBuild)">
```

Comment:
@asbjornu @arturcic  This looks like a problem with the rebase. If you compare this csproj file to @stazz's PR you can see his is different: https://github.com/GitTools/GitVersion/pull/1634/files#diff-bca0d1ca1ea8fbf39936384490e12d86R41